src/transport/BLE.cpp

```diff
@@ -39,9 +39,20 @@ namespace Transport {
 
 BLE::~BLE()
 {
+    ClearState();
+}
+
+void BLE::ClearState()
+{
+    if (mBleLayer)
+    {
+        mBleLayer->CancelBleConnection();
```

Comment:
since CancelConnection returns an error, could we log it?

This will prompt us to implement this in linux at least and maybe others.

Reply:
Also wondering about the name, since it seems as if this 'cancels the current connection'.

Maybe a name of `CancelIncompleteConnection` or similar would be clearer.

Reply:
> since CancelConnection returns an error, could we log it?
> 
> This will prompt us to implement this in linux at least and maybe others.

I have added a log inside `BleLayer.cpp` if the error message is `CHIP_ERROR_NOT_IMPLEMENTED`.